compiler/util/astlocs.cpp

```diff
@@ -57,7 +57,7 @@ int astlocT::compare(const astlocT& other) const {
 void astlocT::convertIdToFileLine(const char*& filename, int& lineno) const {
   if (!this->id_.isEmpty()) {
     // figure out the location from the ID
-    chpl::Location loc = chpl::parsing::locateID(gContext, this->id_);
+    chpl::Location loc = chpl::parsing::locateId(gContext, this->id_);
```

Comment:
I like that you've gone from locateID -> locateId as I'm all for camelizing acronyms\initialisms rather than capitalizing every letter. For my rationale, see the comments I make in the second part of this comments I made in one of Lydia's PRs.

https://github.com/chapel-lang/chapel/pull/17936#issuecomment-864320049

Anyway, given that reasoning should we also rename ID to Id just for consistency's sake?

Reply:
I'm on the fence about renaming ID but I'm sure we don't want it in this PR if we do it.

Reply:
Yeah fair enough. It would be nice if we had some sort of naming convention that we could add to a style guide. I get that this is the sort of things that can end up with endless, and arguably pointless, debate though.